docs/troubleshoot/troubleshoot-apiml-error-codes.md

```diff
@@ -659,7 +727,7 @@ The following error message codes may appear on logs or API responses. Use the f
 
   **Action:**
 
-  Use a valid body in the request. Format of a message: `{validity: int , scopes: [string]}`.
+  Use a valid body in the request. Format of a message: {validity: int , scopes: [string]}.
```

Comment:
Hey @achmelo,
Here is the example where code-blocks might be getting automatically removed.